RecoHI/HiCentralityAlgos/plugins/HiHFFilterProducer.cc

```diff
@@ -0,0 +1,103 @@
+
+#include <memory>
+
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Utilities/interface/StreamID.h"
+
+#include "RecoLocalCalo/CaloTowersCreator/src/CaloTowerCandidateCreator.h"
```

Comment:
can you clarify why this header is needed?

Reply:
Not needed, I remove that on commit f5397ec4d90e6b54c5f6b056650ebb6cb334036c